features/call/impl/src/main/kotlin/io/element/android/features/call/impl/ui/CallScreenPresenter.kt

```diff
@@ -180,6 +180,7 @@ class CallScreenPresenter @AssistedInject constructor(
             urlState = urlState.value,
             webViewError = webViewError,
             userAgent = userAgent,
+            isCallActive = isJoinedCall,
```

Comment:
Have you checked this also works for URL-based calls? I'm not sure `isJoinedCall` is ever set to `true` for those.

Reply:
https://github.com/element-hq/element-x-android/pull/3685/commits/cf80b3b804fe99363d5a988f2295f92ddb04f56a